packages/cadl-ranch-specs/http/azure/resource-manager/models/resources/extension.tsp

```diff
@@ -0,0 +1,174 @@
+import "@typespec/http";
+import "@typespec/rest";
+import "@azure-tools/typespec-azure-core";
+import "@azure-tools/typespec-azure-resource-manager";
+
+using TypeSpec.Http;
+using TypeSpec.Rest;
+using Azure.Core;
+using Azure.ResourceManager;
+using TypeSpec.OpenAPI;
+
+namespace Azure.ResourceManager.Models.Resources;
+
+@resource("extensionResources")
+model RoleAssignment is ExtensionResource<RoleAssignmentProperties> {
```

Comment:
Better to be consistent here, e.g. `@resource("roleAssignments")`

Reply:
Fixed in the new version.